packages/components/src/internal/components/domainproperties/DomainForm.tsx

```diff
@@ -126,13 +126,15 @@ interface IDomainFormInput {
     onChange: (newDomain: DomainDesign, dirty: boolean, rowIndexChange?: DomainFieldIndexChange[]) => any;
     onToggle?: (collapsed: boolean, callback?: () => any) => any;
     panelStatus?: DomainPanelStatus;
+    queryName?: string; // Alternate queryName to use for text choice distinct value query if schema/query not set on domain prop
+    schemaName?: string; // Alternate schemaName to use for text choice distinct value query if schema/query not set on domain prop
```

Comment:
No big deal, but the logic below choose these values first before looking at the domain. Probably intentional, but does the comment want updating?

Reply:
Good call. I should reverse that usage to fall back to this instead.

Reply:
Actually, if a usage is going through the trouble of passing these in, it is for good reason. I'll leave the implementation and update the comment.